_rules/audio-text-alternative-e7aa44.md

```diff
@@ -10,6 +10,11 @@ accessibility_requirements:
     failed: not satisfied
     passed: further testing needed
     inapplicable: further testing needed
+  wcag-technique:G158: # Providing an alternative for time-based media for audio-only content
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: satisfied
```

Comment:
Passed / inapplicable needs to be "further testing needed". There are other ways to embed videos that wouldn't satisfy this technique.